src/CardSubtitle.js

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import elementType from 'prop-types-extra/lib/elementType';
+
+import { createBootstrapComponent } from './ThemeProvider';
+
+class CardSubtitle extends React.Component {
+  static propTypes = {
+    /**
+     * @default 'card-subtitle'
+     */
+    bsPrefix: PropTypes.string.isRequired,
+
+    componentClass: elementType
+  };
+
+  static defaultProps = {
+    componentClass: 'div'
+  };
+
+  render() {
+    const {
+      bsPrefix,
+      className,
+      componentClass: Component,
+      ...props
+    } = this.props;
+
+    const isDefaultComponent = Component === 'div';
```

Comment:
maybe we can define this one like: https://github.com/react-bootstrap/react-bootstrap/blob/v4-support/src/Alert.js#L119-L127 and save some bytes?